### DISCUSSION TITLE
Budbase behind "authenticating" corporate proxy

### DISCUSSION THREAD

**Discussion by pandel** [Budibase/budibase#7171] [Help] [answered]
Hi!

I read all the threads about this problem, but it didn't solve my problem. When I set

```
GLOBAL_AGENT_HTTP_PROXY: http://user:pass@proxy-ip:8080
GLOBAL_AGENT_HTTPS_PROXY: http://user:pass@proxy-ip:8080
GLOBAL_AGENT_NO_PROXY: couchdb-service,app-service,worker-service,minio-service,redis-service,localhost
```

in the worker and app service, I still get no data.

Is there a special syntax for using the whole thing with an authenticating proxy? The examples in https://github.com/gajus/global-agent#runtime-configuration don't talk about it either.

In the meantime, I built the single docker container variant and integrated proxychains (https://salsa.debian.org/debian/proxychains-ng), which is working perfectly, but I'd like to use the official images.

Regards,
Holger

**Comment by melohagan** [comment]
Hey @pandel 

This looks wrong to me: `GLOBAL_AGENT_HTTPS_PROXY: http://user:pass@proxy-ip:8080`

You need to use **https** and the port probably wouldn't be 8080. Maybe 8443.

**Reply by pandel** [reply]
@melohagan That may look wrong to _you_, but it is not! Our proxy is http:// for both, sorry. I can't tell you the correct URL, because it is internal and confidential, but trust me, it is **http** AND **8080**!

BTW, I wouldn't have written it this way if it wasn't like this!

**Reply by shogunpurple** [reply]
Hi @pandel - I don't think an authenticating proxy should make any difference given it's just a URL. What happens when you load the builder - any errors, or anything in your proxy logs that could help us track this down?

**Comment by pandel** [comment, marked as answer]
Solved! Somehow, the password of the proxy user was the showstopper. We changed it and after resetting everything proxy access is working atm.
